src/encoded/ingestion/submission_folio.py

```diff
@@ -95,3 +119,14 @@ def record_results(self, results: dict, validation_summary: list) -> None:
         # be very large, as it is an itemization of each/every object written to the database.
         #
         self.submission.process_standard_bundle_results(results, s3_only=True)
+
+    @staticmethod
+    def _construct_data_file_name_suitable_for_s3(filename: str) -> str:
```

Comment:
Strongly recommend unit testing such a function, if not already tested in snovault